drv/riscv-plic/build.rs

```diff
@@ -0,0 +1,163 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+use std::collections::HashMap;
+use std::env;
+use std::fs::File;
+use std::io::Write;
+use std::path::PathBuf;
+
+task_config::task_config! {
+    ints: &'static [u16],
+    tasks: &'static [&'static str],
+    notification: &'static [u32],
+    base: u32,
+    pbits: u8,
+}
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
I agree. Because the maps can contain anything I'd expect there to be a good bit of weird templating involved, but I can see about adding this in a separate PR.

Reply:
SG.